pkg/generator/ingress_translator.go

```diff
@@ -466,3 +517,24 @@ func domainsForRule(rule v1alpha1.IngressRule) []string {
 	}
 	return domains
 }
+
+func checkCertBundle(certs []byte) error {
+	for block, rest := pem.Decode(certs); block != nil || len(rest) > 0; block, rest = pem.Decode(rest) {
+		if block != nil {
+			switch block.Type {
+			case "CERTIFICATE":
+				_, err := x509.ParseCertificate(block.Bytes)
```

Comment:
Are we planning to do any verification at some point like with this function: https://github.com/istio/istio/blob/master/security/pkg/pki/util/keycertbundle.go#L284 or as soon as we get the cert bundle we assume an external entity is responsible to create valid certs?

Reply:
Yes, we give the CA to envoy, envoy will then use it to verify the upstream TLS server cert.

Reply:
Yeah just wondering if Istio is doing a pre-validation tep.